src/png/mod.rs

```diff
@@ -490,26 +376,35 @@ impl PngData {
         // Go down one step at a time
         // Maybe not the most efficient, but it's safe
         if self.ihdr_data.color_type == ColorType::RGBA {
-            if reduce_rgba_to_grayscale_alpha(self) || reduce_rgba_to_rgb(self) {
+            if reduce_rgba_to_grayscale_alpha(self) {
```

Comment:
Are this and the later `reduce_rgb_to_grayscale` intended to be left as is? Is that part of the aforementioned WIP?

Reply:
That's the WIP part. I'll convert other ones too.